### PR TITLE
Fix merged conflicts

### PR DESCRIPTION
**Pull request by JoshuaStorm**
I accidentally merged master in my branch instead of vice versa, resulting in conflicts being merged.

This PR corrects my mistake.

### CODE REVIEW

File: bin/gltf-pipeline.js [hunk 1]
```diff
@@ -37,17 +29,10 @@ var fileExtension = path.extname(gltfPath);
...
-<<<<<<< HEAD
 var outputPath = defaultValue(argv._[1], defaultValue(argv.o, argv.output));
 var binary = defaultValue(argv.b, defaultValue(argv.binary, false));
 var separate = defaultValue(argv.s, defaultValue(argv.separate, false));
 var separateImage = defaultValue(argv.t, defaultValue(argv.separateImage, false));
-=======
-var outputPath = defaultValue(argv._[1], argv.o);
-var isSeparate = defaultValue(argv.s, false);
-var isBinary = defaultValue(argv.b, false);
-var isQuantized = defaultValue(argv.q, false);
```
Comment (lilleyse) [inline bin/gltf-pipeline.js]: All the quantized related code needs to stay in, but just change the naming like the others.

File: bin/gltf-pipeline.js [hunk 2]
```diff
@@ -63,15 +48,9 @@ if (!defined(outputPath)) {
...
 
 var options = {
-<<<<<<< HEAD
     binary : binary,
     embed : !separate,
     embedImage : !separateImage
-=======
-    isBinary : isBinary,
-    isEmbedded : !isSeparate,
-    isQuantized : isQuantized
```
Comment (lilleyse) [inline bin/gltf-pipeline.js]: Here as well, and change the naming in `processJSONWithExtras` too.

File: bin/gltf-pipeline.js [hunk 3]
```diff
@@ -15,19 +15,11 @@ var readGltf = require('../lib/readGltf');
...
         '  -o --output, output=PATH write optimized glTF to the specified file.\n' +
         '  -b --binary, write binary glTF file.\n' +
         '  -s --separate, writes out separate geometry/animation data files, shader files and textures instead of embedding them in the glTF file.\n' +
         '  -t --separateImage, write out separate textures, but embed geometry/animation data files, and shader files.\n';
-=======
-        '  -i, input=PATH Read unoptimized glTF from the specified file.\n ' +
-        '  -o, output=PATH write optimized glTF to the specified file.\n' +
-        '  -b, write binary glTF file.\n' +
-        '  -s, writes out separate geometry/animation data files, shader files and textures instead of embedding them in the glTF file.\n' + 
-        '  -q, quantize the attributes of this model.\n';
```
Comment (lilleyse) [inline bin/gltf-pipeline.js]: And the flag here.

**Comment by lilleyse** [comment]
`README` still have merge conflicts

**Comment by JoshuaStorm** [comment]
Updated

**Pull request merged**